pkg/languagedetection/privileged_detector.go

```diff
@@ -8,12 +8,20 @@
 package languagedetection
 
 import (
```

Comment:
As we add more imports, this is growing the size of both the process agent and the system probe. Should we consider separating priv vs cli detection logic into separate packages?

Reply:
I think it _should_ be fine for this PR, since all of the imports here are being used elsewhere in the system probe AFAIK. But I do generally agree that we should think about breaking this out. I think this may make a good follow up PR

Reply:
> I think it should be fine for this PR, since all of the imports here are being used elsewhere in the system probe

My concern is that these are used in the system probe and not the process agent, yet still being imported in the process agent

Reply:
Ah, I see. I generally agree, although I still think that this should be a follow up PR since it isn't really related to the purpose of this PR. LMK if you disagree and I'm happy to go ahead and do it.

Reply:
Yep! 👍🏻

Reply:
https://datadoghq.atlassian.net/browse/PROC-3362?atlOrigin=eyJpIjoiNDYzOWNkYWY2YTQ2NGE5ZjgwOTQ5OGFkYWUyNDdhMTUiLCJwIjoiaiJ9